packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -202,8 +202,7 @@ export const messages = {
     'SFDX: ISV Debugger Setup, Step 7 of 7: Converting package: %s',
   isv_debug_bootstrap_processing_package: 'Processing package: %s',
   isv_debug_bootstrap_generate_launchjson: 'Creating launch configuration',
-  isv_debug_bootstrap_open_project:
-    'Opening project in new Visual Studio Code window',
+  isv_debug_bootstrap_open_project: 'Opening project in Visual Studio Code',
```

Comment:
We can't guarantee that we will open the project in a new VSCode window if the user already has that project open. This is working as designed and controlled on VSCode's side.

Reply:
@ekapner Minor wording update for ISV Debugger upon completion in VSCode.

Reply:
I think she's on PTO this week but we can always do a follow up PR once she's back